src/fairchem/core/models/base.py

```diff
@@ -228,68 +229,78 @@ def forward(self, data: Batch) -> dict[str, torch.Tensor]:
         return
 
 
-class HydraInterface(ABC):
-    # a hydra has a backbone and heads
-    @abstractmethod
-    def get_backbone(self) -> BackboneInterface:
-        raise not NotImplementedError
-
-    @abstractmethod
-    def get_heads(self) -> dict[str, HeadInterface]:
-        raise not NotImplementedError
-
-
 @registry.register_model("hydra")
-class HydraModel(nn.Module, GraphModelMixin, HydraInterface):
+class HydraModel(nn.Module, GraphModelMixin):
     def __init__(
         self,
-        backbone: dict,
-        heads: dict,
+        backbone: dict | None = None,
+        heads: dict | None = None,
+        finetune_config: dict | None = None,
         otf_graph: bool = True,
+        pass_through_head_outputs: bool = False,
```

Comment:
Should pass_through_head_outputs be head specific instead of on/off for all heads? Instead of a single bool there would be one per head. Maybe there is an edge case where multiple heads are specified in the the config one which returns multiple values and another that does not.

Reply:
Maybe for now if `pass_through_head_outputs: True` we check that there is only 1 head specified? Or just ignore this edge case for the moment.